pkg/repo/mysql/services_integration_test.go

```diff
@@ -0,0 +1,224 @@
+package mysql
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+	"go.expect.digital/translate/pkg/tracer"
+)
+
+var mysqlRepo *Repo
+
+func TestMain(m *testing.M) {
+	ctx := context.Background()
+
+	viper.SetEnvPrefix("translate_mysql")
+	viper.AutomaticEnv()
+
+	tp, err := tracer.TracerProvider()
+	if err != nil {
+		log.Panicf("set tracer provider: %v", err)
+	}
+
+	conf := &Conf{
+		Host:     viper.GetString("host"),
+		Port:     viper.GetString("port"),
+		User:     viper.GetString("user"),
+		Database: viper.GetString("database"),
+	}
+
+	mysqlRepo, err = NewRepo(WithDBConfig(ctx, conf))
+	if err != nil {
+		log.Panicf("create new repo: %v", err)
+	}
+
+	code := m.Run()
+
+	mysqlRepo.db.Close()
+
+	if err := tp.Shutdown(ctx); err != nil {
+		log.Panicf("tp shutdown: %v", err)
+	}
+
+	os.Exit(code)
+}
+
+func createTestService() *model.Service {
+	return &model.Service{
+		Name: gofakeit.FirstName(),
+		ID:   uuid.New(),
+	}
+}
+
+// insertTestService inserts service to DB.
+func insertTestService(t *testing.T, ctx context.Context, service *model.Service) error {
+	t.Helper()
+
+	_, err := mysqlRepo.db.ExecContext(ctx, `INSERT INTO service (id, name) VALUES (?, ?)`, service.ID, service.Name)
+	if err != nil {
+		return fmt.Errorf("insert test service: %w", err)
+	}
+
+	return nil
+}
+
+func Test_MysqlSaveService(t *testing.T) {
```

Comment:
No need to have Mysql in the test function name. We know it is `mysql` package. Please update all test function names.

```suggestion
func Test_SaveService(t *testing.T) {
```